### PR TITLE
Akamai Security Events Data Connector

### PR DESCRIPTION
**Pull request by socprime**
## Proposed Changes

  - added Akamai Security Events Data Connector

### CODE REVIEW

File: DataConnectors/AkamaiSecurityEvents/Connector_CEF_Akamai.json [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        {
+            "metricName": "Total data received",
+            "legend": "Akamai Events",
+            "baseQuery": "CommonSecurityLog\n| where DeviceVendor == \"Akamai\"\n| where DeviceProduct == \"akamai_siem\""
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Top 10 Countries", 
+            "query": "AkamaiSIEMEvent\n | summarize count() by AkamaiSiemCountry\n | top 10 by count_"
```
Comment (v-jayakal) [inline DataConnectors/AkamaiSecurityEvents/Connector_CEF_Akamai.json:17]: AkamaiSiemCountry is not the column returned in AkamaiSIEMEvent kusto function. Is the query referring to 'SrcGeoCountry'? Can you please correct this query?

File: DataConnectors/AkamaiSecurityEvents/Connector_CEF_Akamai.json [hunk 2]
```diff
@@ -0,0 +1,118 @@
+{
+    "id": "AkamaiSecurityEvents",
+    "title": "Akamai Security Events",
+    "publisher": "Akamai",
+    "descriptionMarkdown": "The Akamai Security Events data connector provides the capability to ingest [Akamai Security Events](https://www.akamai.com/us/en/products/security/) events into Azure Sentinel using syslog. Refer to [Akamai SIEM Integration documentation](https://developer.akamai.com/tools/integrations/siem) for more information.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on Kusto Function to work as expected. Follow the steps to use this Kusto Function alias **AkamaiSIEMEvent** in queries and workbooks. [Follow steps to get this Kusto Function>](https://github.com/Azure/Azure-Sentinel/blob/master/Parsers/Akamai/AkamaiSIEMEvent.txt)  ",
```
Comment (v-jayakal) [inline DataConnectors/AkamaiSecurityEvents/Connector_CEF_Akamai.json:6]: Please use 'https://aka.ms/sentinel-akamaisecurityevents-parser' for the Kusto Function URL.

File: DataConnectors/AkamaiSecurityEvents/Connector_CEF_Akamai.json [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "Akamai Events",
```
Comment (v-jayakal) [inline DataConnectors/AkamaiSecurityEvents/Connector_CEF_Akamai.json:10]: Please use 'AkamaiSecurityEvents' for legend value as that is DATATYPE_NAME. Refer https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_CEF_template.json.

**Review by v-jayakal — CHANGES_REQUESTED** [review]

File: DataConnectors/AkamaiSecurityEvents/Connector_CEF_Akamai.json (hunk 1 above)
Comment (socprime) [reply DataConnectors/AkamaiSecurityEvents/Connector_CEF_Akamai.json]: done

File: DataConnectors/AkamaiSecurityEvents/Connector_CEF_Akamai.json (hunk 2 above)
Comment (socprime) [reply DataConnectors/AkamaiSecurityEvents/Connector_CEF_Akamai.json]: done

File: DataConnectors/AkamaiSecurityEvents/Connector_CEF_Akamai.json (hunk 3 above)
Comment (socprime) [reply DataConnectors/AkamaiSecurityEvents/Connector_CEF_Akamai.json]: done

**Review by v-jayakal — APPROVED** [review]

**Review by nazang — APPROVED** [review]

**Pull request merged**